src/Microsoft.Extensions.Configuration/IncludeConfigurationProvider.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Configuration
+{
+    public class IncludeConfigurationProvider : ConfigurationProvider
```

Comment:
Is the naming brainstorming still open? I have a slight preference for `Import()` and `ImportedConfigurationProvider`. Otherwise I think this class should probably be called `IncludedConfigurationProvider` (notice the "d" that is currently missing).

Caveat: English is my second language